### PR TITLE
upgrade to stein

### PR DESCRIPTION
**Pull request by AlvaroVega**
- [x] Test all APIs

### CODE REVIEW

File: keystone_scim/contrib/scim/core.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from keystone.common import extension
+#from keystone.common import extension
```
Comment (fgalan) [inline keystone_scim/contrib/scim/core.py:21]: Leftover?

File: keystone_scim/contrib/scim/core.py [hunk 2]
```diff
@@ -36,8 +36,8 @@
     ]
 }
 
-extension.register_admin_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
-extension.register_public_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
+#extension.register_admin_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
+#extension.register_public_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
```
Comment (fgalan) [inline keystone_scim/contrib/scim/core.py:40]: Leftovers?

File: keystone_scim/contrib/scim/core.py (hunk 1 above)
Comment (AlvaroVega) [reply keystone_scim/contrib/scim/core.py]: fixed in https://github.com/telefonicaid/fiware-keystone-scim/pull/63/commits/4823e20f7cb3887260bc88b7f2591b2bfd31dda6

File: keystone_scim/contrib/scim/core.py (hunk 2 above)
Comment (AlvaroVega) [reply keystone_scim/contrib/scim/core.py]: fixed in https://github.com/telefonicaid/fiware-keystone-scim/pull/63/commits/4b29f070f4eec20f8ee2bdad2810b01d8cd92198

**Pull request merged**